src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -311,6 +311,125 @@ def k8s_install_cli(client_version='latest', install_location=None):
+    --service-principal is not specified, the secret is auto-generated for you
+    and stored in ${HOME}/.azure/ directory.
+    :type client_secret: str
+    """
+    from subprocess import PIPE, Popen
+    helm_not_installed = "Helm not detected, please verify if it is installed."
+    image_tag = 'latest'
+    url_chart = chart_url
+    # Check if Helm is installed locally
+    try:
+        Popen(["helm"], stdout=PIPE, stderr=PIPE)
+    except OSError:
+        raise CLIError(helm_not_installed)
+    # Validate if the RG exists
+    groups = _resource_client_factory().resource_groups
+    # Just do the get, we don't need the result, it will error out if the group doesn't exist.
+    rgkaci = groups.get(resource_group)
+    # Auto assign the location
+    if location is None:
+        location = rgkaci.location  # pylint:disable=no-member
+    # Get the credentials from a AKS instance
+    _, browse_path = tempfile.mkstemp()
+    aks_get_credentials(client, resource_group, name, admin=False, path=browse_path)
+    subscription_id = _get_subscription_id()
+    dns_name_prefix = _get_default_dns_prefix(connector_name, resource_group, subscription_id)
+    # Ensure that the SPN exists
+    principal_obj = _ensure_service_principal(service_principal, client_secret, subscription_id,
+                                              dns_name_prefix, location, connector_name)
+    client_secret = principal_obj.get("client_secret")
+    service_principal = principal_obj.get("service_principal")
+    # Get the TenantID
+    profile = Profile()
+    _, _, tenant_id = profile.get_login_credentials()
+    # Check if we want the windows connector
+    if os_type.lower() in ['windows', 'both']:
+        # The current connector will deploy two connectors, one for Windows and another one for Linux
+        image_tag = 'canary'
+    logger.warning('Deploying the aci-connector using Helm')
+    try:
+        subprocess.call(["helm", "install", url_chart, "--name", connector_name, "--set", "env.azureClientId=" +
+                         service_principal + ",env.azureClientKey=" + client_secret + ",env.azureSubscriptionId=" +
+                         subscription_id + ",env.azureTenantId=" + tenant_id + ",env.aciResourceGroup=" + rgkaci.name +
+                         ",env.aciRegion=" + location + ",image.tag=" + image_tag])
+    except subprocess.CalledProcessError as err:
+        raise CLIError('Could not deploy the ACI Chart: {}'.format(err))
+
+
+def k8s_uninstall_connector(client, name, connector_name, resource_group,
+                            graceful=False, os_type='Linux'):
+    """Undeploy the ACI-Connector from an AKS cluster.
+    :param name: The name of the AKS cluster. The name is case insensitive.
+    :type name: str
+    :param resource_group: The name of the resource group where the AKS cluster is deployed.
+    :type resource_group: str
+    :param connector_name: The name for the ACI Connector
+    :type connector_name: str
+    """
```

Comment:
You can still just remove this block of comments.
Everything should be defined in params.py already.

Reply:
Got it, let me move all of it in the `params.py` file to make it cleaner.